pressbooks/pressbooks
# Pb 1831 footnote issue refactor

https://github.com/pressbooks/pressbooks/issues/1831

## How to test
- Create a footnote in a chapter with different elements like lists, images, quotes, etc
- Create more than 1 footnote in the same paragraph

Check in web and PDF export footnotes have the expected behavior.
